libs/booking/feature-seat-listing/src/lib/booking-feature-seat-listing.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'aap-listing-info',
+  template: `
+    <h1>
+      SeatListing
+    </h1>
+  `
+})
+export class BookingFeatureSeatListingComponent implements OnInit {
```

Comment:
I'm wondering what would be the best name here.

If it's a _page_ component with nested routed components (it has a `<router-outlet>` in its template), this name is a good candidate.

If not, we might just call it `SeatListingComponent`. I only know of the entry point Angular modules needing the long names.

What do you think, @NachoVazquez?

Reply:
Yes, I agree, I'm not sure if the Nx CLI generated such a long name, but either way, I would only add the full scope name path to the module. Any other component, service, etc. gets messy, having long names.

I'm going to experiment with the CLI to see what's the default behavior.